### PR TITLE
Big Cleaning

### PR DESCRIPTION
**Pull request by zayk00v**

### CODE REVIEW

File: contracts/Farming/LiquidityHandler.sol [hunk 1]
```diff
@@ -103,10 +103,11 @@ contract LiquidityHandler is
         __UUPSUpgradeable_init();
 
         require(_multiSigWallet.isContract(), "Handler: Not contract");
+        require(_exchangeAddress.isContract(), "Handler: Not contract");
         exchangeAddress = _exchangeAddress;
         _grantRole(DEFAULT_ADMIN_ROLE, _multiSigWallet);
-        _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
         _grantRole(UPGRADER_ROLE, _multiSigWallet);
+        _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
```
Comment (pentatonictritones) [inline contracts/Farming/LiquidityHandler.sol:110]: This was added for tests only so can we remove this

File: hardhat.config.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ const config: HardhatUserConfig = {
     settings: {
       optimizer: {
         enabled: true,
```
Comment (pentatonictritones) [inline hardhat.config.ts:20]: Let's clean this up a little bit.

I think : 

1. Add all the networks we are using 
2. Add the updated env.example to the PR

**Review by pentatonictritones — CHANGES_REQUESTED** [review]
Just these two small things. The rest looks good because it is mostly shuffling around files for cleaning that we have previously reviewed.

**Review by pentatonictritones — APPROVED** [review]

**Review by 0xtuytuy — APPROVED** [review]
LGTM

**Pull request merged**